@here/harp-mapview/lib/MapView.ts

```diff
@@ -526,19 +525,6 @@ export interface MapViewOptions extends TextElementsRendererOptions {
      */
     maxFps?: number;
 
-    /**
```

Comment:
Please add information about tghis breaking API change to the release note draft:
https://github.com/heremaps/harp.gl/releases/tag/untagged-b803742d56359e72466c